2024.md

```diff
@@ -3,7 +3,11 @@
 ## January (2)
 * [Coding Authentication? Stop now and let Choreo manage it.](https://dev.to/suvink/coding-authentication-stop-now-and-let-choreo-manage-it-4ofa) by [Suvin Nimnaka](https://dev.to/suvink)
 * [CRYSTALS Kyber : The Key to Post-Quantum Encryption](https://medium.com/@hwupathum/crystals-kyber-the-key-to-post-quantum-encryption-3154b305e7bd) by [Udara Pathum](https://medium.com/@hwupathum)
+
+# 2024 Vlogs
```

Comment:
Instead of repeating 2024 in the markdown file name and each subtitle, shall we do something as follows?

title: 2024.md

## Blogs

## Vlogs

Reply:
@VivekVinushanth WDYT ?

Reply:
+1. Vlogs are rare and hence let us move them below. Added a section below with https://github.com/wso2/iam-blogs/pull/328

Reply:
PR updated